### DISCUSSION TITLE
Memory maxes out, install crashes, any macOS.

### DISCUSSION THREAD

**Discussion by bucketsofmail** [quickemu-project/quickemu#1521] [Quickemu]
Hello all, 
After getting the first image, I format the drive with Mac Disk Manager as instructed then move on to the Mac OS Re-install. While the install is running, it runs for a few minutes fine, starting around an hour and counting down. After a bit, the time remaining jumps way up, my actual (bare metal) machine's memory slowly maxes out, the entire host OS locks up and the install crashes. Machine goes back to normal. When I load Quickemu back up, the only drive symbol present is EFI, no more way to get back in and try the install again. I have to use QuickGUI to delete the entire VM. 

I've tried Catalina, Big-Sur and Mojave. Tried to adjust to different ram sizes 4, 6, 8, different  disk sizes and numbers of cores 4, 8, 16 and 32. Always the same result. I am able to run macOS in VirtualBox, but I want to try something else. 

Specs: 
Bare Metal Machine: 2017 Macbook Pro 13", touchbar, 4 port, 8G mem, Intel Iris 650 IGPU, Intel i5 processor
Host OS: Linux Mint 22 Cinnamon

**Comment by ghost** [comment]
Take a look at #1589 .

Long story short, you may need to make a swap partition from some of your unallocated partition. If you're in a situation where you probably don't have enough disk space anyway, there's another option to use even a USB thumbdrive as a partition if you use the program "gparted".

- WARNING - 
Carefully read everything and understand exactly what you're doing before using gparted because it has the ability to completely erase and modify partitions, including your main OS parititons!

https://gparted.org/download.php

This thread may also be helpful to you
https://askubuntu.com/questions/349156/how-to-use-hard-disk-as-ram-like-in-windows
